am-lib/src/main/java/uk/gov/hmcts/reform/amlib/db/migration/V3__Create_default_roles_config_tables.sql

```diff
@@ -0,0 +1,47 @@
+CREATE TYPE securityclassification AS ENUM ('Public', 'Private', 'Restricted');
+
+CREATE TABLE roles (
+  role_name varchar(100) PRIMARY KEY,
+  role_type VARCHAR (50) NOT NULL,
+  security_classification securityclassification NOT NULL
+);
+
+CREATE TABLE services (
+  service_name varchar(100) PRIMARY KEY,
+  service_description varchar(250)
+);
+
+CREATE TABLE resources (
+  service_name varchar(100) NOT NULL,
+  resource_type varchar(100) NOT NULL,
+  resource_name varchar(100) NOT NULL,
+  PRIMARY KEY (service_name, resource_type, resource_name),
+  CONSTRAINT resources_service_name_fkey FOREIGN KEY (service_name)
+    REFERENCES services (service_name)
+    ON UPDATE NO ACTION ON DELETE NO ACTION
+);
+
+CREATE TABLE resource_attributes (
+  service_name varchar(100) NOT NULL,
+  resource_type varchar(100) NOT NULL,
+  resource_name varchar(100) NOT NULL,
+  attribute varchar(250) NOT NULL,
+  default_security_classification securityclassification NOT NULL,
+  PRIMARY KEY (service_name, resource_type, resource_name, attribute),
+  CONSTRAINT resource_attributes_fkey FOREIGN KEY (service_name, resource_type, resource_name)
+    REFERENCES resources (service_name, resource_type, resource_name)
+    ON UPDATE NO ACTION ON DELETE NO ACTION
+);
+
+CREATE TABLE default_permissions_for_roles (
+  service_name varchar(100) NOT NULL,
```

Comment:
missing FK to table 'resources'

Reply:
In fact it should be a FK to `resource_attribute`